Incorrect exports in @opencode-ai/sdk 1.2.9 package.json causing import failures

### Description


Package.json exports say:

```
{
  import: ./dist/index.js,
  types: ./dist/index.d.ts
}
```

But dist/ actually contains:

```
dist/
├── src/
│   ├── index.js
│   ├── index.d.ts
│   └── ...
└── tsconfig.tsbuildinfo
```

So dist/index.js and dist/index.d.ts don't exist at the expected locations — the real files are at dist/src/index.js and dist/src/index.d.ts.

using bun

related: https://github.com/anomalyco/opencode/issues/11366

### Plugins

_No response_

### OpenCode version

1.2.9

### Steps to reproduce

1. install `@opencode-ai/sdk`
2. try and import `import { createOpencode } from '@opencode-ai/sdk';`

```
error: Cannot find module '@opencode-ai/sdk' from '/home/jhsu/code/opencode-test/src/index.ts'

Bun v1.3.9 (Linux x64)
```

### Screenshot and/or share link

_No response_

### Operating System

Archlinux

### Terminal

ghostty

--- a/.gitignore
+++ b/.gitignore
@@ -27,3 +27,4 @@ target
 opencode-dev
 logs/
 *.bun-build
+tsconfig.tsbuildinfo

--- a/packages/sdk/js/package.json
+++ b/packages/sdk/js/package.json
@@ -13,15 +13,15 @@
     "./client": "./src/client.ts",
     "./server": "./src/server.ts",
     "./v2": {
-      "types": "./dist/src/v2/index.d.ts",
+      "types": "./dist/v2/index.d.ts",
       "default": "./src/v2/index.ts"
     },
     "./v2/client": {
-      "types": "./dist/src/v2/client.d.ts",
+      "types": "./dist/v2/client.d.ts",
       "default": "./src/v2/client.ts"
     },
     "./v2/gen/client": {
-      "types": "./dist/src/v2/gen/client/index.d.ts",
+      "types": "./dist/v2/gen/client/index.d.ts",
       "default": "./src/v2/gen/client/index.ts"
     },
     "./v2/server": "./src/v2/server.ts"

--- a/packages/sdk/js/tsconfig.json
+++ b/packages/sdk/js/tsconfig.json
@@ -7,7 +7,8 @@
     "declaration": true,
     "moduleResolution": "nodenext",
     "lib": ["es2022", "dom", "dom.iterable"],
-    "composite": true
+    "composite": true,
+    "rootDir": "src"
   },
   "include": ["src"]
 }

